.travis.yml

```diff
@@ -52,6 +52,7 @@ matrix:
 branches:
   only:
     - master
+    - impl-EZP-24434-aa-core-2
```

Comment:
Removed in https://github.com/ezsystems/ezpublish-kernel/commit/c7eaa711d0f3afaa1ee2258951528e9f821c15b8